features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenPresenter.kt

```diff
@@ -163,11 +167,18 @@ class CallScreenPresenter @AssistedInject constructor(
                 is CallScreenEvents.SetupMessageChannels -> {
                     messageInterceptor.value = event.widgetMessageInterceptor
                 }
+                is CallScreenEvents.OnWebViewError -> {
+                    if (!ignoreWebViewError) {
+                        webViewError = event.description.orEmpty()
```

Comment:
Should this `webViewError` be refreshed if `ignoreWebViewError` becomes true after?

Reply:
There is no mechanism currently to load again the page if the network is back. So not null `webViewError` is sort of a final state.